shared/api-ref-ex.asciidoc

```diff
@@ -3,9 +3,24 @@
 ++++
 <titleabbrev>Sample</titleabbrev>
 ++++
+////
+This section includes a brief, 1-2 sentence description
+followed immediately by a basic request example.
+DO NOT include the response.
+////
 
 Creates a cool thing.
 
+[source,js]
```

Comment:
I intentionally avoided creating a heading for the basic request example.

@KOTungseth Let me know if this looks alright from a Kibana POV.

I can add a comment about making this example request optional if needed.

Reply:
Made this optional but recommended with 7800035 and 85c39d0.